### PR TITLE
test: impl Display trait on structs used in callbacks for logging

### PR DESCRIPTION
**Pull request by Manuthor**

### CODE REVIEW

File: src/structs.rs [hunk 1]
```diff
@@ -435,14 +436,64 @@ impl Serializable for IndexedValue {
...
+    fn into_iter(self) -> Self::IntoIter {
+        self.0.into_iter()
+    }
+}
+
+impl<const UID_LENGTH: usize> Display for Uids<UID_LENGTH> {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let mut output = String::new();
+        for uid in self.clone() {
+            output = format!("uid: {}\n{output}", STANDARD.encode(uid));
```
Comment (tbrezot) [inline src/structs.rs:471]: Is this really what you want ? I mean it seems to me that only the last UID is displayed, but maybe there is something I didn't get.

Comment (Manuthor) [reply src/structs.rs:471]: it also takes the `output` variable for each element of the list

Comment (tbrezot) [reply src/structs.rs:471]: Ah okay, I got it, you include `output` in the `format!()`... why don't you simply extend the output with the new `format` ?

Comment (Manuthor) [reply src/structs.rs]: done in https://github.com/Cosmian/findex/pull/65/commits/9b7dde9771647d197ad4474ccdec55e0be60bd8a

**Pull request merged**